<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1026</width>
    <height>611</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="glView" name="openGLWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>601</width>
      <height>511</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_back">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>450</y>
      <width>231</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>BACK TO 2007</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineFileName">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>601</width>
      <height>21</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>230</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Transformations</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>40</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Informations</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>30</y>
      <width>371</width>
      <height>151</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>220</y>
      <width>371</width>
      <height>201</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>250</y>
      <width>341</width>
      <height>161</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>x</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>y</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>z</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Rotation</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBox_xRotObj">
         <property name="minimum">
          <number>-180</number>
         </property>
         <property name="maximum">
          <number>180</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBox_yRotObj">
         <property name="minimum">
          <number>-180</number>
         </property>
         <property name="maximum">
          <number>180</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBox_zRotObj">
         <property name="minimum">
          <number>-180</number>
         </property>
         <property name="maximum">
          <number>180</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Translation</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBox_xTranslation">
         <property name="minimum">
          <number>-360</number>
         </property>
         <property name="maximum">
          <number>360</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBox_yTranslation">
         <property name="minimum">
          <number>-360</number>
         </property>
         <property name="maximum">
          <number>360</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBox_zTranslation">
         <property name="minimum">
          <number>-360</number>
         </property>
         <property name="maximum">
          <number>360</number>
         </property>
         <property name="stepType">
          <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Scale</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBox_xScaling">
         <property name="suffix">
          <string> /100</string>
         </property>
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>1000</number>
         </property>
         <property name="singleStep">
          <number>10</number>
         </property>
         <property name="value">
          <number>100</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBox_yScaling">
         <property name="suffix">
          <string> /100</string>
         </property>
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>1000</number>
         </property>
         <property name="singleStep">
          <number>10</number>
         </property>
         <property name="value">
          <number>100</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBox_zScaling">
         <property name="suffix">
          <string> /100</string>
         </property>
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>1000</number>
         </property>
         <property name="singleStep">
          <number>10</number>
         </property>
         <property name="value">
          <number>100</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>661</x>
      <y>50</y>
      <width>341</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Vertexes</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_19">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineVertexes">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Facets</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_20">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineFacets">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>530</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Status:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_recordingGIF">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>530</y>
      <width>261</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1026</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_File"/>
    <addaction name="actionSave_As"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionSettings"/>
   </widget>
   <widget class="QMenu" name="menuImport">
    <property name="title">
     <string>Import</string>
    </property>
    <addaction name="actionImport_to_JPEG"/>
    <addaction name="actionImport_to_BMP"/>
    <addaction name="actionImport_to_GIF"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuImport"/>
  </widget>
  <action name="actionOpen_File">
   <property name="text">
    <string>Open File...</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save As...</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionImport_to_JPEG">
   <property name="text">
    <string>Import to JPEG</string>
   </property>
  </action>
  <action name="actionImport_to_BMP">
   <property name="text">
    <string>Import to BMP</string>
   </property>
  </action>
  <action name="actionImport_to_GIF">
   <property name="text">
    <string>Import to GIF</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>glView</class>
   <extends>QOpenGLWidget</extends>
   <header>glview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
